Add Proxy support for websocket

oldshuren opened the pull request:
This pull request add proxy support for socket.io client behind a proxy server.

The proxy configuration is getting from the system. The proxy server must support the CONNECT method.

When determined that there is proxy setting, instead of opening a network stream to the remote websocket server, it opens a network stream to the proxy server, then issuing an HTTP CONNECT request. If the proxy server responds with 200 OK, it treats this network stream as a normal stream to the remote websocket server.

nuclearace commented:
All pull request for WebSocket should be made at https://github.com/daltoniam/Starscream. Also you shouldn't change podspec to point to yours, I would create a new version and push it.

oldshuren commented:
Ok, thanks!

I modified podspec because 'path' and 'branch' attribute in my Podfile
didn't work. But even I changed the podspec, it still didn't work. I'll
changed it back.

On Tue, May 31, 2016 at 12:11 PM, Erik Little <notifications@github.com>
wrote:

> All pull request for WebSocket should be made at
> https://github.com/daltoniam/Starscream. Also you shouldn't change
> podspec to point to yours, I would create a new version and push it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/socketio/socket.io-client-swift/pull/380#issuecomment-222738670>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHRzfF-u6m77eZ1vn5ApKBElpLBBf6GPks5qHF3LgaJpZM4IqPzb>
> .
>

neiker commented:
Here's the PR on Starscream repo:
https://github.com/daltoniam/Starscream/pull/547
